### ISSUE TITLE
Remove stack attack branch names from repository

### ISSUE THREAD

**Issue by saphal1998** [taneliang/stack-attack#60]
Is there any relevance to us having stack attack branch names in our repository post landing? I was wondering that this gets ugly very fast if we choose to display these branch names even after the merge. I think there should be a `hook` of sorts that deletes the stack attack branches post landing.

It shouldn't impact the PR navigation stack (in the description) post landing though as the PR numbers would still remain the relevant commit hashes should be found.

<img width="1157" alt="Screenshot 2020-08-16 at 12 52 58 PM" src="https://user-images.githubusercontent.com/31125345/90329121-8917b000-dfbf-11ea-8b42-2d903ef2d9e5.png">

**Comment by saphal1998** [comment]
Let's just take this on Discord.

**Comment by taneliang** [comment]
> Let's just take this on Discord.

😆 

It's definitely ugly if we keep them around, but I think they should be deleted by the user. On the remote, GitHub can automatically delete branches when a PR is merged. On the local checkout, users already need a way to delete branches corresponding to old branches that have been deleted, so they should already have a way to delete branches that have been landed. This is my solution: https://git.sr.ht/~el/unixconfig/tree/bee20f6c/.config/fish/config.fish#L107

So TL;DR I don't think it's within our scope to solve this, especially for launch. We can certainly consider adding a delete branch command in the future though, I think that'll be helpful for anyone using Git.
